typescript/src/client/solana/solana.types.ts

```diff
@@ -23,11 +26,25 @@ export type SolanaClientInterface = Omit<
   signMessage: (options: SignMessageOptions) => Promise<SignatureResult>;
   signTransaction: (options: SignTransactionOptions) => Promise<SignatureResult>;
 };
+
 /**
- * A Solana account.
+ * A base Solana account.
  */
 export type Account = OpenAPISolanaAccount;
 
+/**
+ * A Solana account with actions.
+ */
+export type SolanaAccount = Prettify<OpenAPISolanaAccount & AccountActions>;
```

Comment:
nit: i think it would be better to define this type in a new file `accounts/solana/types.ts`